.travis.yml

```diff
@@ -19,5 +20,9 @@ matrix:
       env: TOXENV=flake8
 install:
   - pip install tox-travis
+  - npm install -g grunt-cli
+  - npm install grunt
+  - npm install grunt-contrib-qunit
 script:
   - tox
+  - grunt test
```

Comment:
It wouldn't be a big impact to runtime, but it's not ideal if we can avoid it. I've asked @nicoddemus to enable this repository with CircleCI so we can move the JS tests there. Until then, you should be able to enable it for your fork and experiment.